docker/Dockerfile

```diff
@@ -161,21 +153,19 @@ ARG DISPLAY
 ARG XAUTHORITY
 
 #Environment setup
-ENV http_proxy=$http_proxy
-ENV https_proxy=$https_proxy
```

Comment:
if proxy is not set inside container image, I assume we need to find a different way to enable proxy for kubernetes setup which doesn't use docker-compose.